### DISCUSSION TITLE
Issue getting helper script

### DISCUSSION THREAD

**Discussion by JOPY13** [Guilouz/Creality-Helper-Script-Wiki#657] [Issues]
Keep getting this after I had to do a reset when trying to git script.. worked fine before and the other K1 gits it, not a total noob but limited with linux.

root@K1-A121 /root [#] git clone https://github.com/Guilouz/Creality-Helper-Scri
pt.git /usr/data/helper-script
Cloning into '/usr/data/helper-script'...
fatal: unable to access 'https://github.com/Guilouz/Creality-Helper-Script.git/': SSL certificate problem: certificate is not yet valid

**Comment by JOPY13** [comment]
Bah... got the launcher running but still getting errors.... and yes, ran this also 

git config --global http.sslVerify false

  Are you sure you want to install Fluidd ? (y/n): y

Info: Downloading Fluidd file...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: certificate is not yet valid
More details here: https://curl.haxx.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
root@K1-A121 /root [#]

**Comment by luxxseed** [comment]
Hello,
you only have to set the right date and time like this:
date -s '2014-12-25 12:34:56'.

**Comment by JOPY13** [comment]
Thank you, haven't tried that yet.. will give that try.

**Comment by JOPY13** [comment]
Thanks again for the response and info... That did it.
